eox_tenant/api/v1/viewsets.py

```diff
@@ -0,0 +1,34 @@
+from eox_tenant.api.v1.serializers import (MicrositeSerializer,
```

Comment:
bad sort, sort in this way 
Future
Python Standard Library
Third Party
Current Python Project

we don't have an specific way to this but I prefer the following way 
![image](https://user-images.githubusercontent.com/36200299/80085633-1c633c00-851e-11ea-92ad-bc0b9d67d92b.png)